cmd/kube_storage_monitor/app/server.go

```diff
@@ -14,12 +14,15 @@ import (
 )
 
 type monitorOpt struct {
-	kube_storage_types []string
+	kube_storage_types  []string
+	enable_node_watcher bool
 }
 
 func (mo *monitorOpt) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&mo.kube_storage_types, "kube-storage-types", mo.kube_storage_types, ""+
 		"kube-storage-types is the backend storage drivers type, such as: local_pv, cephfs, rbd... ")
+	fs.BoolVar(&mo.enable_node_watcher, "enable-node-watcher", mo.enable_node_watcher, ""+
```

Comment:
is node watcher only used for local PVs?

Reply:
yes, networked storage PV will not be effected by node failure.